docker/irods_clients/scripts/irods_client_wrapper.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
When running as `iinit`, this script shows the password in plain text:
```
WARNING: Error 25 disabling echo mode. Password will be displayed in plaintext.
Enter your current iRODS password:
```
Is this expected?